chains/celo/listener.go

```diff
@@ -33,3 +44,28 @@ func (l *listener) start() error {
 func (l *listener) close() {
 	l.conn.Close()
 }
+
+func (l *listener) getTransactionBlockHash(hash common.Hash) (blockHash common.Hash, err error) {
+
+	receipt, err := l.conn.Client().TransactionReceipt(context.Background(), hash)
+	if err != nil {
+		return hash, fmt.Errorf("unable to get BlockHash: %s", err)
+	}
+	return receipt.BlockHash, nil
+}
+
+func (l *listener) getBlockTransactionsByHash(hash common.Hash) (txHashes []common.Hash, txRoot common.Hash, err error) {
```

Comment:
I think it would be more clear to name this `getTransactionsFromBlockHash` and hash -> `blockHash`